wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/RealGrpcCall.kt

```diff
@@ -139,7 +139,11 @@ internal class RealGrpcCall<S : Any, R : Any>(
     val result = grpcClient.newCall(method, requestMetadata, requestBody, timeout)
     this.call = result
     if (canceled) result.cancel()
-    (timeout as ForwardingTimeout).setDelegate(result.timeout())
+    // If the timeout doesn't have a deadline or timeout, then the user
+    // didn't set the timeout on this Call manually.
```

Comment:
> then the user didn't set the timeout on this Call manually

Does this imply there are other ways to set it?

Reply:
Well it gets set by default to `ForwardingTimeout(Timeout())` when it's initialized.